make/Makefile.dev

```diff
@@ -38,6 +38,12 @@ create-cr:
 build-operator-image:
 	operator-sdk build $(DEVOPSCONSOLE_OPERATOR_IMAGE)
 
+.PHONY: push-operator-image
+## Push the operator container image to a container registry
+push-operator-image: build-operator-image
+	docker login -u $(QUAY_USERNAME) -p $(QUAY_PASSWORD) "quay.io"
+	docker push $(DEVOPSCONSOLE_OPERATOR_REGISTRY)
```

Comment:
@baijum Is your PR dependent on another PR? where is DEVOPSCONSOLE_OPERATOR_REGISTRY defined?

Reply:
I had another PR which introduced it. I merged today morning.

Reply:
Fixed 50c2e205deee1acbb6e505943c9f4743443d3a29